model/src/main/proto/exploration.proto

```diff
@@ -108,14 +108,29 @@ message Solution {
   string interaction_id = 1;
   // Flag that is true if correct_answer is the only correct answer of the question.
   bool answer_is_exclusive = 2;
-  string correct_answer = 3;
+
+  CorrectAnswer correct_answer = 3;
```

Comment:
Add documentation for each of these fields

Reply:
Done.